__tests__/nc-news5.test.js

```diff
@@ -67,4 +67,41 @@ describe("/api/articles", () => {
         });
       });
   });
+  test("200 should return articles filtered by a topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("topic", "mitch");
+        });
+      });
+  });
+  test("should return  if an invalid query is made", () => {
+    return request(app)
+      .get("/api/articles?topic=xylophone")
+      .expect(200)
```

Comment:
This should really be a 404 - the topic doesn't exist so therefore cannot be found

Reply:
I have tried to implement this, but as the api is correct it is not giving me an error but only an empty array. Which is why I am checking the length with 200 code status.